glean-core/android/src/main/java/mozilla/telemetry/glean/Glean.kt

```diff
@@ -53,6 +53,7 @@ data class GleanTimerId internal constructor(internal val id: Long)
 open class GleanInternalAPI internal constructor () {
     companion object {
         private const val LOG_TAG: String = "glean/Glean"
+        private const val LANGUAGE_BINDING_NAME: String = "Kotlin"
```

Comment:
This will result in the User-Agent being `Glean/31.x.x (Kotlin on Android)`, right?

Reply:
Yes. That means it would be different from before, but also means now this header will look the same for all platform + binding combinations.

Reply:
Cool, let's just update the docs then!